### PR TITLE
fix(magicStringTable4): EVAL-361 move magic string in the correct file and replace it

### PR DESCRIPTION
**Pull request by simonchaval**
## Describe your changes
Move magic string in the correct file and make changes in the code.
## Issue ticket number and link
EVAL-361 : https://entsupport.gdapublic.fr/browse/EVAL-361
## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [ ] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: src/main/java/fr/openent/competences/service/impl/DefaultNiveauDeMaitriseService.java [hunk 1]
```diff
@@ -81,7 +82,7 @@ public void getNiveauDeMaitriseofCycle(Long Cycle, Handler<Either<String, JsonAr
         JsonArray values = new fr.wseduc.webutils.collections.JsonArray();
 
         query.append("SELECT niv1.libelle, niv1.ordre, niv1.couleur couleurDefault, niv1.id_cycle ")
-                .append("FROM notes.niveau_competences niv1 ")
+                .append("FROM " + Field.NOTES_TABLE + "." + Field.NIVEAU_COMPETENCES_TABLE + " niv1 ")
```
Comment (lapeyronnie) [inline src/main/java/fr/openent/competences/service/impl/DefaultNiveauDeMaitriseService.java:85]: attention, c'est le schéma notes et pas la table

File: src/main/java/fr/openent/competences/service/impl/DefaultNiveauDeMaitriseService.java [hunk 2]
```diff
@@ -94,11 +95,11 @@ public void getNiveauDeMaitriseofClasse(String idClasse, Handler<Either<String,
         StringBuilder query = new StringBuilder();
         JsonArray values = new fr.wseduc.webutils.collections.JsonArray();
 
-        query.append("SELECT niveau_competences.libelle, niveau_competences.ordre, ")
-                .append(" niveau_competences.couleur couleurDefault, niveau_competences.id_cycle  ")
-                .append(" FROM notes.niveau_competences")
+        query.append("SELECT " + Field.NIVEAU_COMPETENCES_TABLE + ".libelle, " + Field.NIVEAU_COMPETENCES_TABLE + ".ordre, ")
+                .append(" " + Field.NIVEAU_COMPETENCES_TABLE + ".couleur couleurDefault, " + Field.NIVEAU_COMPETENCES_TABLE + ".id_cycle  ")
+                .append(" FROM " + Field.NOTES_TABLE + "." + Field.NIVEAU_COMPETENCES_TABLE)
```
Comment (lapeyronnie) [inline src/main/java/fr/openent/competences/service/impl/DefaultNiveauDeMaitriseService.java:100]: attention, c'est le schéma notes et pas la table

File: src/main/java/fr/openent/competences/service/impl/DefaultNoteService.java [hunk 3]
```diff
@@ -232,7 +233,7 @@ public void listNotesParDevoir(Long devoirId, Handler<Either<String, JsonArray>>
                 .append(table_appreciation).append(" WHERE ").append(note_id_devoir).append(" = ")
                 .append(appreciation_id_devoir).append(" AND ").append(note_id_eleve).append(" = ")
                 .append(appreciation_id_eleve).append(" ) ").append("ORDER BY 2")
-                .append(") AS res, notes.devoirs WHERE res.id_devoir = devoirs.id");
+                .append(") AS res, " + Field.NOTES_TABLE + ".devoirs WHERE res.id_devoir = devoirs.id");
```
Comment (lapeyronnie) [inline src/main/java/fr/openent/competences/service/impl/DefaultNoteService.java:236]: attention, c'est le schéma notes et pas la table

**Comment by simonchaval** [comment]
Wrong method, now gonna use file the existant "Field.java" with magic string key to update in the continuity of the project.
cf : [This PR](https://github.com/OPEN-ENT-NG/competences/pull/127)

**Pull request closed without merging**